src/ai/backend/client/func/vfolder.py

```diff
@@ -350,9 +350,9 @@ async def _download_file(
                                                                 chunk_size
                                                             )
                                                         except asyncio.TimeoutError:
-                                                            raise TryAgain
```

Comment:
Couldn't we just simply raise a "root" exception instead of specifying `from None` always for brevity?